Verify macros documentation for v8 (#2225)
umbraco/UmbracoDocs: Reference/Templating/Macros/index.md

For example imagine adding an Image Gallery within a rich text editor, and at the point of insertion 'picking' the images to display.

![Insert Image Carousel](images/image-carousel-macro-v8.png)

Define the parameters

![Define the parameters](images/image-carousel-macro-parameter-v8.png)

Using in a Rich Text Area

A Rich Text Editor should be enabled with macros in the toolbar to allow inserting macros.

![Enable macro toolbar in Rich Text Area](images/rte-macro.png)

Rich Text Area with macro toolbar option

![Macro toolbar option in Rich Text Area](images/rte-macro-toolbar.png)

Insert the macro into a Rich Text Area

![Insert the macro into a Rich Text Area](images/pick-images-for-macro-example-v8.png)

The same implementation logic can be used in lots of different places on your site, and the editor can customise the output by choosing different parameters.

## Implementing a Macro

> Macro can be implemented using a MVC Partial View - (aka Partial View Macros), [Details of implementing a Partial View Macro](Partial-View-Macros/index.md).It uses the exact same syntax and objects as [MVC views](Mvc/index.md).

Reviewer: ```suggestion
Macros can be implemented using an MVC Partial View - [Partial View Macros](Partial-View-Macros/index.md). It uses the exact same syntax and objects as [MVC views](Mvc/index.md).
```
